modules/ROOT/pages/introduction-to-tiny-spellchecker.adoc

```diff
@@ -22,7 +22,6 @@ include::partial$misc/purchase-premium-plugins.adoc[]
 
 To enable the {productname} Enterprise Spellchecking plugin with xref:editor-and-features.adoc[{cloudname}]:
 
-. If configured, disable the `+spellchecker+` plugin provided with {productname}, by removing it from the `+plugins+` list.
 . Add `+tinymcespellchecker+` to the `+plugins+` list.
```

Comment:
Same issue here now... this is a list with only a single item and I think it could be collapsed.